R/brace_linter.R

```diff
@@ -8,9 +8,14 @@
 #'  - Closing curly braces in `if` conditions are on the same line as the corresponding `else`.
 #'  - Either both or neither branch in `if`/`else` use curly braces, i.e., either both branches use `{...}` or neither
 #'    does.
-#'  - Functions spanning multiple lines use curly braces.
+#'  - Function bodies are wrapped in curly braces.
 #'
-#' @param allow_single_line if `TRUE`, allow an open and closed curly pair on the same line.
+#' @param allow_single_line If `TRUE`, allow an open and closed curly pair on the same line.
+#' @param function_bodies Whether to require function bodies to be wrapped in curly braces. One of
```

Comment:
"Whether" kind of implies a binary choice

```suggestion
#' @param function_bodies Character string governing when function bodies without curly braces should generate lints. One of
```

Reply:
Also, as @AshesITR points out, I think we should be careful here to specify "function body" for all the rules, since a function _definition_ includes the parameters and those can take up many lines even if the function itself doesn't (not that we expect to see that except in strange cases).

Reply:
Addressed in https://github.com/r-lib/lintr/pull/2240/commits/01bb196ed89797d089b5d19a96152a878617c57e. I've opted for a shorter description without the param type ("character string") since it should become clear from the second sentence (and we do `match.arg(function_bodies)` anyways).